workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import timedelta
+import logging
+import json
+from dapr.ext.workflow import DaprWorkflowContext, WorkflowActivityContext
+# , when_any
+from model import InventoryItem, Notification, InventoryRequest, OrderPayload, OrderResult, PaymentRequest, InventoryResult
+from dapr.clients import DaprClient
+from util import get_address
+
+store_name = "statestore-actors"
+
+address = get_address()
+
+def order_processing_workflow(ctx: DaprWorkflowContext, orderPayload: OrderPayload):
+    logging.basicConfig(level=logging.INFO)
+    order_id = ctx.instance_id
+    yield ctx.call_activity(notify_activity, input=Notification(message=f'Received order {order_id} for {orderPayload.quantity} {orderPayload.item_name} at ${orderPayload.total_cost} !'))
+    result = yield ctx.call_activity(verify_inventory_activity, input=InventoryRequest(request_id=order_id, item_name=orderPayload.item_name, quantity=orderPayload.quantity))
+    if not result.success:
+        yield ctx.call_activity(notify_activity, input=Notification(message=f'Insufficient inventory for {orderPayload.item_name}!'))
+        return OrderResult(processed=False)
+    
+    if orderPayload.total_cost > 50000:
+        yield ctx.call_activity(requst_approval_activity, input=orderPayload)
+        approval_flag = ctx.wait_for_external_event("manager_approval")
```

Comment:
Since we're returning a `Task` and not a boolean flag directly, I suggest we name this variable `approval_task`.
```suggestion
        approval_task = ctx.wait_for_external_event("manager_approval")
```